tutorials/forward/25_automated_coreg.py

```diff
@@ -107,3 +109,12 @@
 #           matched to a subject's head digitization. When scaling is desired,
 #           a scaled surrogate MRI should be created using
 #           :func:`mne.scale_mri`.
+#
+# .. LINKS
+#
+# .. _`pathlib.Path`: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module
```

Comment:
If we did link to this it would be better to do it via intersphinx like
```
:class:`python:pathlib.Path`
```
```suggestion
```

Reply:
Sounds good, the >80 character link was causing flake8 errors anyway.